readme.md

```diff
@@ -0,0 +1,27 @@
+# Welcome 
+
+Welcome to the Courtgrabber Documentation public repository. These files have been formatted according to the [MkDocs](https://www.mkdocs.org/) theme and are best viewed on an MkDocs site. We have not yet found a permanent hosting service, but we hope to make the move soon. In the meantime, you can view the files by downloading MkDocs, cloning this repo, and running the built-in MkDocs dev server to view the site locally.
+
+## Installing MkDocs
+
+Run the following command from your command line to install MkDocs:
+
+```
+pip install mkdocs
+```
+
+If you encounter any issues, refer to the [MkDocs Installation Guide](https://www.mkdocs.org/user-guide/installation/) for more detailed instructions.
+
+## Creating your Courtgrabber project
+
+[Fork](https://docs.github.com/en/get-started/quickstart/fork-a-repo) and [clone](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository) this repo to your local machine. Your project directory should contain the `mkdocs.yml` configuration file and a `docs` directory with several Markdown files and an `images` subdirectory.
+
+## Viewing the Courtgrabber MkDocs site
+
+From your command line, navigate to your new project directory and then run the following command:
+
+```
+mkdocs serve
+```
+
+With the server running, open `http://127.0.0.1:8000/` in your browser to view Courtgrabber's documentation.
```

Comment:
Nit: You could consider leaving `http://127.0.0.1:8000/` unformatted so it renders as a link.  That makes it just a bit easier for a user reading this documenting while setting things up to visit the docs site in their web browser by simply clicking the link.